rero_ils/modules/patrons/api.py

```diff
@@ -434,9 +439,22 @@ def remove_role(self, role_name):
 
     @property
     def patron(self):
-        """Patron property shorcut."""
+        """Patron property shortcut."""
         return self.get('patron', {})
 
+    @property
+    def expiration_date(self):
+        """Shortcut to find the patron expiration date."""
+        date_string = self.patron.get('expiration_date')
+        if date_string:
+            return datetime.strptime(date_string, '%Y-%m-%d')
+
+    @property
+    def is_expired(self):
+        """Check if patron expiration date is obsolete."""
+        expiration_date = self.expiration_date
+        return expiration_date and datetime.now() > expiration_date
```

Comment:
Do we have to work with UTC times?

Reply:
`expiration_date` stored into the patron resource doesn't have any time (it's just YYYY-MM-DD).
Using `datetime.now()`, I'm sure tu use timestamp with server localized timezone.
So I think check is consistant and correct here